org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Nisida/ImportNisida.java

```diff
@@ -351,14 +350,16 @@ protected static void processUnit(final String line, final NisidaLoadState statu
 		final String platformName = tokens[1];
 
 		// FIND THE PLATFORM.
-		TrackWrapper track = (TrackWrapper) status.getLayers().findLayer(platformName);
-		if (track == null) {
-			track = new TrackWrapper();
-			track.setColor(DebriefColors.RED);
-			track.setName(platformName);
-			status.getLayers().addThisLayer(track);
+		if (status.getLayers() != null) {
```

Comment:
This class only makes sense if there is a layers object, so we shouldn't have to check for the presence of the layer object.

If this non-null check is because we pass in `null` for layers, then I think it would be better to fix the unit test, please.

Reply:
Fixed at https://github.com/debrief/debrief/pull/4913/commits/027ed1e5f6c6d1bc73a19add6dd38bc89bf9d17d